src/main/java/com/example/springresttask/domain/dto/EmailDto.java

```diff
@@ -0,0 +1,23 @@
+package com.example.springresttask.domain.dto;
+
+
+import javax.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class EmailDto {
+
+    @NotNull
+    private String recipientName;
```

Comment:
If recipient name is an email, will the following values be appropriate: "", " ", "     "? NotNull is not the best way to handle some use cases for strings. There is a better annotation that includes more cases